### ISSUE TITLE
tidysynth 0.1.0: function spelling error grab_signficance

### ISSUE THREAD

**Issue by SeanRichterWalsh** [edunford/tidysynth#22]
Probably a typo?

**Comment by michalovadek** [comment]
I think this is fixed now, but there are still quite a few typos in the documentation. I'd recommend running `devtools::spell_check()` to catch some of them.

This is just nitpicking though - I actually came here to say that this package is absolutely wonderful. I think other R event study packages (e.g. did) could benefit from copying your very intuitive API

**Comment by edunford** [comment]
Absolutely a typo! Thanks to you both for checking on this and the `devtools::spell_check()` is a great suggestion @michalovadek! I wrote this package in a week (preparing for a lecture I was giving) and I think it shows in all the spelling mishaps!

Luckily this change was fixed in the development version in May (see https://github.com/edunford/tidysynth/issues/11). I've dragged my feet on pushing an update to CRAN but will do so later this week to carry over a few minor bug fix and spelling corrections that I've made. Thanks to you both for using the package!
